hedera-node/src/main/java/com/hedera/services/ledger/accounts/BackingNfts.java

```diff
@@ -60,9 +60,7 @@ public MerkleUniqueToken getImmutableRef(NftId id) {
 	public void put(NftId id, MerkleUniqueToken nft) {
 		final var key = fromNftId(id);
 		final var currentNfts = delegate.get();
-		if (!currentNfts.containsKey(key)) {
-			currentNfts.put(key, nft);
-		}
+		currentNfts.computeIfAbsent(key, k -> nft);
```

Comment:
I would slightly prefer to use the more verbose `containsKey` to avoid creating the lambda in the second argument. ..not sure if it matters though.

Reply:
Modified to be using as before